worker/src/core/config.ts

```diff
@@ -310,6 +315,23 @@ const config: Config<ConfigSchema> = convict({
     default: '.gov.sg',
     env: 'SHOW_MASTHEAD_DOMAIN',
   },
+  phonebookContactPref: {
+    enabled: {
+      doc: 'Enable display of phonebook contact preferences',
+      default: false,
+      env: 'SHOW_PHONEBOOK_CONTACT_PREF',
+    },
+    url: {
+      doc: 'Phonebook uri to fetch contact preferences',
+      default: 'phonebook.postman.gov.sg',
+      env: 'PHONEBOOK_URL',
+    },
+    apiKey: {
+      doc: 'API key for Phonebook contact preferences api',
+      default: '',
+      env: 'PHONEBOOK_API_KEY',
+    },
+  },
```

Comment:
after reviewing the use case, i think we should use `required-string` format here